### ISSUE TITLE
I cannot reproduce Form Factors results from BBJW:2018A with b-to-gamma-l-nu_TEST.cc parameters.

### ISSUE THREAD

**Issue by edvelasquezal** [eos/eos#717]

**Comment by mreboud** [comment]
Thanks for rising this issue @edvelasquezal. Can you provide more information? What exactly are you trying to do?

**Comment by edvelasquezal** [comment]
Sorry, I didn't say more indeed.

I am trying to obtain the results from BBJW:2018A, i.e., the plots for the form factors in terms of the photon energy. How ever the parameters used to test this case are different from the ones in BBJW:2018A. The you obtain kind of agree but the use of different parameters make it confusing.We are studying this with Danny to check with Phillip about these values.

**Comment by mreboud** [comment]
I am not sure I understand why this is an issue. `b-to-gamma-l-nu_TEST.cc` uses a specific set of parameter to test the implementation, in principle this test could be completely random.

My question is, can you reproduce BBJW:2018A is you use the same parameters as in BBJW:2018A?

**Comment by dvandyk** [comment]
This is still being investigated, but summary from the dev meeting today:
@pluegh suggests that simply adjusting the input values to exactly match what is used in Beneke 2018 will yield tests that should pass against the Beneke 2018 reference results
- Most salient difference: ``mu_0 = 1.5`` in the test case, but should be ``mu_0 = 1.0``.
- Probe if changing the factorization scales ``mu_h1`` and ``mu_h2`` makes a difference here.

To be further investigated.

**Comment by edvelasquezal** [comment]
We now manage to reproduce the results of BBJW:2018A.

The discrepancy for the perturbative term of the FFS would come either from the Radiative Factor or the LCDA. The former was OK while the latter had the discrepancy.

We needed to correct the set of parameters used in eos for it to match BBJW:2018A.

Using RunDec, we checked that the alpha_s(mu_0) value quoted in Table 1 of BBJW2018 is incompatible with the measured alpha_s(MZ) = 0.1180 at three-loop order. Artifically lowering alpha_s(mu_0) from ~0.46 to ~0.34 removes most of the gap we see between the EOS and the BBJW2018 curves!

This exploration inspired us to create switch options in the code such that one can choose in the FFs observables the different contributions to handle and the loop order for alpha_s(mu).
